lib/middleware.js

```diff
@@ -512,6 +516,22 @@ module.exports = function middlewareConstructor(env) {
           res.json({"status": 200});
         });
       };
+    },
+
+    /**
+     * If there's an oauth token, decrypt it and set the
+     * local user object.
+     */
+    decryptToken: function(req, res, next) {
+      if (!req.session.token) {
```

Comment:
`cookieSession` is a global middleware that creates a session for every request if it doesn't exist. Something must have gone really really wrong for it not to exist here.